package.json

```diff
@@ -24,7 +24,11 @@
 		"create": "composer install && yarn workspaces run build && yarn run install:env",
 		"wp-env": "wp-env",
 		"lint:php": "composer run lint",
-		"format:php": "composer run format"
+		"format:php": "composer run format",
+		"lint:css": "yarn workspace wporg-learn-plugin lint:css && yarn workspace wporg-learn-theme lint:css",
+		"lint:js": "yarn workspace wporg-learn-plugin lint:js",
+		"watch:theme": "yarn workspace wporg-learn-theme start",
+		"watch:plugin": "yarn workspace wporg-learn-plugin start"
```

Comment:
Shortcuts for the scripts listed https://github.com/WordPress/Learn/blob/trunk/readme.md#linting and https://github.com/WordPress/Learn/blob/trunk/readme.md#development